src/Services/FHIR/FhirResourcesService.php

```diff
@@ -16,6 +16,8 @@
 use OpenEMR\FHIR\R4\FHIRResource\FHIRBundle\FHIRBundleLink;
 use OpenEMR\FHIR\R4\PHPFHIRResponseParser;
 
+require_once('_rest_config.php');
```

Comment:
are we sure this relative path will always work. seems better to use an absolute path here:
```
require_once(__DIR__ . '/../../../_rest_config.php');
```

Reply:
You know I had a heck of a time getting that to work. I will add the DIR and path info in and retest

Reply:
Should be able to put this in the OpenEMR namespace and path your way to a use statement, true?

Reply:
@dixonwhitmire just leave this and i'll move and refactor this.